test/deploy/linux/nginx/deploy-application/.net/debian/roles/onbeforestart/tasks/main.yml

```diff
@@ -0,0 +1,78 @@
+---
+- debug:
+    msg: Install .NET App
+
+- name: Set default is_selfcontained (default to false)
+  set_fact:
+    is_selfcontained: "false"
+  when: is_selfcontained is undefined
+
+- name: Set default download url for self contained web app
+  set_fact:
+    web_app_url: "https://virtuoso-testing.s3-us-west-2.amazonaws.com/selfcontained-net5webapplication.zip"
+  when: web_app_url is undefined and (is_selfcontained|bool)
+
+- name: Set default download url for framwork dependent web app
+  set_fact:
+    web_app_url: "https://virtuoso-testing.s3-us-west-2.amazonaws.com/frameworkdependent-net5webapplication.zip"
```

Comment:
We need to consolidate those zips on the same S3 account we already use for the other tests: https://s3.console.aws.amazon.com/s3/buckets/open-install-library-artifacts?region=us-west-2

Not sure if you have access to our AWS account, otherwise @gabeobrien can

Reply:
I don't appear to have the necessary permissions to add new content to the S3 bucket.  We do want to consolidate the zips, though.